tests/Kron.py

```diff
@@ -37,14 +39,111 @@
 
 #-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
 
-class KronPhotometryTestCase(unittest.TestCase):
+def makeGalaxy(width, height, flux, a, b, theta, dx=0.0, dy=0.0, xy0=None, xcen=None, ycen=None):
```

Comment:
It seems like we're adding a lot of duplicate code here for no obvious reason. This `makeGalaxy` seems to be very similar to code in `KronPhotometryTestCase.makeAndMeasure()`. You could probably make a convincing argument that refactoring the code, breaking it up into smaller functions, isn't a bad idea, but this duplication seems to be uncalled for.